src/watchlists/WatchList.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  View,
+  Image,
+  Text,
+  TouchableHighlight,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import WatchListItem from './WatchListItem';
+import chevron from '../../assets/icons/watchList/chevron.png';
+import Card from '../common/Card';
+import styles from './styles';
+
+export default function watchlists({ name }) {
+  const stocks = useSelector((state) => state.watchlists.stocks);
+  const [flag, setFlag] = useState(true);
```

Comment:
- flag? What kind of flag? Chinese flag? Hungarian flag? How about `isOpen`?
- boolean values should have `is`, `has`, `should` prefixes